.github/workflows/release.yml

```diff
@@ -279,11 +279,23 @@ jobs:
           # Add additional line before the content
```

Comment:
The code changes look correct and well-commented. It seems like the author is modifying the `RELEASE_NOTES` variable by removing certain characters, replacing markdown links with confluence links, replacing usernames with links to GitHub profiles, and converting newlines to `<br/>` tags.